cis_policies_generator/README.md

```diff
@@ -1,53 +1,77 @@
-    "impact": "None",
-    "references": [
-        "https://kubernetes.io/docs/admin/kube-apiserver/"
-    ],
-    "section": "Control Plane Node Configuration Files",
-    "benchmark": {
-        "name": "CIS Kubernetes V1.23",
-        "version": "v1.0.1"
-    }
-}
+Usage: cis-policies-generator [options]
+
+CIS Benchmark parser CLI
+
+Options:
+  -V, --version             output the version number
+  -t, --templates           generate csp rule templates and place them in the integration dir
+  -b, --benchmark <string>  benchmark to be used for the rules template generation (default: "cis_k8s")
+  -m, --rulesMeta           generate rules metadata for any provided benchmark in the input dir
+  -h, --help                display help for command
+
+    Example calls:
+        $ npm start -- -m
+        $ npm start -- -t -b 'cis_eks'
 ```
 
-There's also a `combined.json` file that will hold all rules from all benchmarks in the following format:
+
+Example serialization of a single rule:
 
 ```
-{
-    "policies": {
-        "CIS_Amazon_Elastic_Kubernetes_Service_(EKS)_Benchmark_v1.1.0": {
-            "2.1.1": {
-                "id": "3a52f937-3893-55ad-a557-a8aaa29d9500",
-                "name": "Enable audit Logs",
-                "rule_number": "2.1.1",
-                ...
-            },
-            "2.1.2": {
-                ...
-            }
-        },
-        "CIS_Kubernetes_V1.20_Benchmark_v1.0.1": {
-            "1.1.1": {
-                ...
-            }
-        }
-    }
-}
+metadata:
+  id: 883ab83b-8dbc-5072-aef7-0f4c4a7f4048
+  name: Ensure that the API server pod specification file permissions are set to
+    644 or more restrictive
+  profile_applicability: "* Level 1 - Master Node"
+  description: Ensure that the API server pod specification file has permissions
+    of `644` or more restrictive.
+  rationale: The API server pod specification file controls various parameters
+    that set the behavior of the API server. You should restrict its file
+    permissions to maintain the integrity of the file. The file should be
+    writable by only the administrators on the system.
+  audit: >-
+    Run the below command (based on the file location on your system) on the
+    Control Plane node. For example,
+
+
+    ```
+
+    stat -c %a /etc/kubernetes/manifests/kube-apiserver.yaml
+
+    ```
+
+
+    Verify that the permissions are `644` or more restrictive.
+  remediation: >
+    Run the below command (based on the file location on your system)
+    on the Control Plane node. For example,
+
+
+    ```
+
+    chmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml
+
+    ```
+  impact: None
+  references:
+    - https://kubernetes.io/docs/admin/kube-apiserver/
+  tags:
+    - CIS
+    - Kubernetes
+    - CIS 1.1.1
+    - Control Plane Node Configuration Files
+  section: Control Plane Node Configuration Files
+  benchmark:
+    name: CIS Kubernetes V1.23
+    version: v1.0.1
+    id: cis_k8s_v1.23
```

Comment:
Changing the Id also means that when we use the script to generate we have to make sure we do the required changes in Kibana UI (I think there will be some).